autopush/web/registration.py

```diff
@@ -71,7 +74,19 @@ def convert_chid(self, data):
 class TokenSchema(SubInfoSchema):
     """Filters allowed values from body data"""
     token = fields.Str(allow_none=True)
-    # Temporarily allow 'aps' definition data for iOS.
+
+
+def validate_token(val):
```

Comment:
There's also a RegExp validator in there, if we have some idea what they could be. Is it possible for a token to be nested structure like a dict?